composer.json

```diff
@@ -9,8 +9,8 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.3 || ^3.0 || ^4.0"
+        "php": ">=7.1.3",
+        "symfony/event-dispatcher": "^4.3"
```

Comment:
Do I understand correctly that we cannot add `|| ^5.0` because the [`EventDispatcherInterface` contract](https://github.com/symfony/symfony/blob/5.0/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php) changes the argument order for `dispatch()`? If so, will compatibility with `^5.0` require a follow-up PR to bump the requirement and change this library's `dispatch()` signature to match exactly?

Reply:
You're right. I think we should generally use `Symfony\Contracts\EventDispatcher\EventDispatcherInterface`.